src/task-2/index.ts

```diff
@@ -0,0 +1,33 @@
+import csv from "csvtojson";
+import { createReadStream, createWriteStream, existsSync, mkdirSync } from "fs";
+import path from "path";
+import stream from "stream";
+import { promisify } from "util";
+
+const pipeline = promisify(stream.pipeline);
+
+const initialMemoryUsed = process.memoryUsage().heapUsed / 1024 / 1024;
```

Comment:
Why do you need memory calculation? If you wanted to prove that not the whole file was uploaded to memory, then it won't work because you measure memory after operation is finished & not in process. The initial task was just to use streaming instead of reading file methods

Reply:
Removed it